appscale/tools/admin_api/handler.py

```diff
@@ -0,0 +1,157 @@
+""" Represents an Admin API UrlMap resource. """
+
+import re
+
+from appscale.tools.custom_exceptions import AppEngineConfigException
+
+DELTA_REGEX = r'([0-9]+)([DdHhMm]|[sS]?)'
+
+EXPIRATION_RE = r'\s*({})(\s+{})*\s*'.format(DELTA_REGEX, DELTA_REGEX)
+
+
+class Handler(object):
+  """ Represents an Admin API UrlMap resource. """
+  API_FIELDS = {
+    'secure': 'securityLevel',
+    'login': 'login',
+    'auth_fail_action': 'authFailAction',
+    'redirect_http_response_code': 'redirectHttpResponseCode'
+  }
+
+  API_VALUES = {
+    'securityLevel': {'optional': 'SECURE_OPTIONAL',
+                      'never': 'SECURE_NEVER',
+                      'always': 'SECURE_ALWAYS'},
+    'login': {'optional': 'LOGIN_OPTIONAL',
+              'required': 'LOGIN_REQUIRED',
+              'admin': 'LOGIN_ADMIN'},
+    'authFailAction': {'redirect': 'AUTH_FAIL_ACTION_REDIRECT',
+                       'unauthorized': 'AUTH_FAIL_ACTION_UNAUTHORIZED'}
```

Comment:
should redirectHttpResponseCode be here also, i.e.

```
    'redirectHttpResponseCode': {301: 'REDIRECT_HTTP_RESPONSE_CODE_301',
                                 302: 'REDIRECT_HTTP_RESPONSE_CODE_302',
                                 303: 'REDIRECT_HTTP_RESPONSE_CODE_303',
                                 307: 'REDIRECT_HTTP_RESPONSE_CODE_307'}
```

Reply:
Yes. Good catch.